sass/includes/_accordion.scss

```diff
@@ -0,0 +1,52 @@
+.accordion {
+    $root: &;
+
+    &.is-open {
+        #{$root}__icon {
+            transform: rotate(180deg);
+        }
+
+        #{$root}__button {
+            margin-bottom: 10px;
+        }
+
+        #{$root}__content {
+            display: block;
+        }
+    }
+
+    &.is-closed {
+        #{$root}__content {
+            display: none;
+        }
+    }
+
+    &__button {
+        background: none;
+        border: 0;
+        display: flex;
+        align-items: center;
+        transition: color 0.2s ease;
+        text-decoration: none;
+        width: 100%;
+
+        &:hover {
+            cursor: pointer;
+        }
+
+        /* stylelint-disable */
+        &:focus {
+            outline: 0; // focus style implemented directly on text instead
```

Comment:
I don't see any focus state when the accordions are closed

Reply:
Resolved, thank you for flagging!